pkg/cas/ipfs/ipfs_test.go

```diff
@@ -74,7 +74,7 @@ func TestRead(t *testing.T) {
 		}))
 		defer ipfs.Close()
 
-		cas := New(ipfs.URL)
+		cas := New(ipfs.URL, false)
```

Comment:
missing test with true

Reply:
Done. I also noticed that the "success" test was using a mocked server. Since I already went to the work of using a real IPFS server in the `cas_test.go` tests, I updated it to use a real IPFS server as well.